prow/cluster/ghproxy.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: ${NAMESPACE}
+  name: ghproxy
+  labels:
+    app: ghproxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ghproxy
+  template:
+    metadata:
+      labels:
+        app: ghproxy
+    spec:
+      tolerations:
+      - key: "node-role.kubernetes.io/master"
+        operator: "Exists"
+        effect: "NoSchedule"
+      nodeSelector:
+        kubernetes.io/hostname: prow-master-03 # This could be a proper label such as `role: ghproxy`
```

Comment:
master 3 is already running the bazel-cache- maybe master 1?

Reply:
is this a problem? A quick `oc describe node` showed that master3 is the most "free" node.

Reply:
actually we should probably start running most of prow on the master nodes